adapters/handlers/rest/embedded_spec.go

```diff
@@ -3480,6 +3489,12 @@ func init() {
     }
   },
   "parameters": {
+    "CommonClassParameterQuery": {
+      "type": "string",
+      "description": "Class parameter specifies the class from which to query objects",
+      "name": "class",
```

Comment:
hmm, I think that we agreed that the parameter name should be `classname` not `class` am I right?